src/CairoMakie.jl

```diff
@@ -531,15 +531,41 @@ function draw_plot(scene::Scene, screen::CairoScreen, primitive::Combined)
 end
 
 function draw_plot(screen::CairoScreen, scene::Scene)
+
+    # get the parent area to correct for its pixel size when translating
+    parent_area = if isnothing(scene.parent)
```

Comment:
There is: https://github.com/JuliaPlots/AbstractPlotting.jl/blob/master/src/scenes.jl#L199